### DISCUSSION TITLE
Infinite Reverse Z projection for JoltViewer

### DISCUSSION THREAD

**Discussion by peter1745** [jrouwe/JoltPhysics#1245] [Ideas]
Hi, I've been utilizing JoltViewer here and there for a little while now, and I really like having a tool that be used to visualize the physics simulation.
But one major problem I have with it is the far clipping plane being very close to the camera position, which can make it hard to see what's happening in large scenes. It would probably be worth using an infinite reverse z projection to deal with this.
While I know that the best approach is most likely to integrate Jolts debug renderer into your own rendering and simply overlay physics information on top of the normal render, it is nice to simply use the `DebugRendererRecorder` and then view it in JoltViewer, which is mainly why I'm suggesting this.
Would've love to hear any thoughts on this 🙂

**Comment by jrouwe** [comment]
Hello,

That is a good idea! I went ahead and implemented it in #1246.
